library/src/interactor_impl.cxx

```diff
@@ -745,76 +753,186 @@ bool interactor_impl::triggerCommand(std::string_view command)
-  this->addBinding("T", ModifierKeys::NONE, "toggle render.effect.tone_mapping");
-  this->addBinding("E", ModifierKeys::NONE, "toggle render.show_edges");
-  this->addBinding("X", ModifierKeys::NONE, "toggle interactor.axis");
-  this->addBinding("G", ModifierKeys::NONE, "toggle render.grid.enable");
-  this->addBinding("N", ModifierKeys::NONE, "toggle ui.filename");
-  this->addBinding("M", ModifierKeys::NONE, "toggle ui.metadata");
-  this->addBinding("Z", ModifierKeys::NONE, "toggle_fps");
-  this->addBinding("R", ModifierKeys::NONE, "toggle render.raytracing.enable");
-  this->addBinding("D", ModifierKeys::NONE, "toggle render.raytracing.denoise");
-  this->addBinding("V", ModifierKeys::NONE, "toggle_volume_rendering");
-  this->addBinding("I", ModifierKeys::NONE, "toggle model.volume.inverse");
-  this->addBinding("O", ModifierKeys::NONE, "toggle model.point_sprites.enable");
-  this->addBinding("U", ModifierKeys::NONE, "toggle render.background.blur");
-  this->addBinding("K", ModifierKeys::NONE, "toggle interactor.trackball");
-  this->addBinding("F", ModifierKeys::NONE, "toggle render.hdri.ambient");
-  this->addBinding("J", ModifierKeys::NONE, "toggle render.background.skybox");
-  this->addBinding("L", ModifierKeys::NONE, "increase_light_intensity");
-  this->addBinding("L", ModifierKeys::SHIFT, "decrease_light_intensity");
-  this->addBinding("H", ModifierKeys::NONE, "toggle ui.cheatsheet");
-  this->addBinding("Question", ModifierKeys::ANY, "print_scene_info");
-  this->addBinding("1", ModifierKeys::ANY, "set_camera front");
-  this->addBinding("3", ModifierKeys::ANY, "set_camera right");
-  this->addBinding("4", ModifierKeys::ANY, "roll_camera -90");
-  this->addBinding("5", ModifierKeys::ANY, "toggle scene.camera.orthographic");
-  this->addBinding("6", ModifierKeys::ANY, "roll_camera 90");
-  this->addBinding("7", ModifierKeys::ANY, "set_camera top");
-  this->addBinding("9", ModifierKeys::ANY, "set_camera isometric");
-  this->addBinding(F3D_EXIT_HOTKEY_SYM, ModifierKeys::NONE, "stop_interactor");
-  this->addBinding("Return", ModifierKeys::NONE, "reset_camera");
-  this->addBinding("Space", ModifierKeys::NONE, "toggle_animation");
-  this->addBinding("Drop", ModifierKeys::NONE, "add_files");
+  this->addBinding("W", ModifierKeys::NONE, "cycle_animation", std::bind(docStrVal, "Cycle animation", animStr));
+  this->addBinding("C", ModifierKeys::NONE, "cycle_coloring field", std::bind(docStrVal, "Cycle point/cell data coloring", pointCellStr));
+  this->addBinding("S", ModifierKeys::NONE, "cycle_coloring array", std::bind(docStrVal, "Cycle array to color with", arrayStr));
+  this->addBinding("Y", ModifierKeys::NONE, "cycle_coloring component", std::bind(docStrVal, "Cycle component to color with", compStr));
+  this->addBinding("B", ModifierKeys::NONE, "toggle ui.scalar_bar", std::bind(docStrVal, "Toggle the scalar bar display",
+    [&]() -> std::string { return docTgl(opts.ui.scalar_bar);}));
```

Comment:
why aren't you using std::bind here?

Reply:
Well, there is nothing to bindm there is no args to this method.

Reply:
Wouldnt that bind that parameter to the current value of `opts.ui.scalar_bar` ? 

Maybe std::bind is confusing after all :)